packages/salesforcedx-vscode-core/src/messages/i18n.ts

```diff
@@ -254,6 +254,7 @@ export const messages = {
   telemetry_legal_dialog_message:
     'You agree that Salesforce Extensions for VS Code may collect usage information, user environment, and crash reports for product improvements. Learn how to [opt out](%s).',
   telemetry_legal_dialog_button_text: 'Read more',
+  telemetry_internal_user_message: 'We\'re collecting information on the usage of Einstein for Developers and sending it to AppInsights to help us understand how our internal developers use it. We want to gather data on developer adoption to improve our products and services. Thanks for being a part of this process!',
```

Comment:
Sorry this has been rolling for a while.  This is DocWriter approved correct?

Reply:
I believe so, we had Sonal give us the message on the story. However, the telemetry is recorded for all extensions if you are internal. It might be good to circle back with her on the message.

Reply:
@sbudhirajadoc @AnanyaJha Can you review the above message.  Two thoughts on it. 1. This isn't E4D specific it's for all SF extensions that use our telemetry (The more I think about it the more E4D seems wrong here, they may not even have E4D installed). 2.  Something about the three sentences feel disconnected.

Reply:
Is this more accurate -- 'We're collecting information on the usage of our extensions and sending it to AppInsights to help us understand how our internal developers use them. We want to gather data on developer adoption to improve our products and services. Thanks for being a part of this process!

@gbockus-sf @AnanyaJha